File: code-samples/meetings/quick-start.cs
using System;
using System.Threading.Tasks;
using RingCentral;

namespace Create_Meeting
{
  class Program
  {
    static RestClient restClient;
    static async Task Main(string[] args)
    {
      restClient = new RestClient(
      Environment.GetEnvironmentVariable("RC_CLIENT_ID"),
      Environment.GetEnvironmentVariable("RC_CLIENT_SECRET"),
      Environment.GetEnvironmentVariable("RC_SERVER_URL"));
      await restClient.Authorize(Environment.GetEnvironmentVariable("RC_JWT"));
      await create_meeting();
    }
    static private async Task create_meeting()
    {
      var parameters = new MeetingRequestResource();
      parameters.topic = "Test Meeting";
      parameters.meetingType = "Instant";
      parameters.allowJoinBeforeHost = true;
      parameters.startHostVideo = true;
      parameters.startParticipantsVideo = false;

      var resp = await restClient.Restapi().Account().Extension().Meeting().Post(parameters);
      Console.WriteLine("Start Your Meeting: " + resp.links.startUri);
      Console.WriteLine("join the Meeting: " + resp.links.joinUri);
    }
  }
}

File: code-samples/meetings/quick-start.php
<?php 
require('vendor/autoload.php');
$dotenv = Dotenv\Dotenv::createImmutable(__DIR__ . '/../');
$dotenv->load();

$rcsdk = new RingCentral\SDK\SDK( $_ENV['RC_CLIENT_ID'],
                                  $_ENV['RC_CLIENT_SECRET'],
                                  $_ENV['RC_SERVER_URL'] );
$platform = $rcsdk->platform();
$platform->login( [ "jwt" => $_ENV['RC_JWT'] ] );

$params = array(
    'topic' => 'Test Meeting',
    'meetingType' => 'Instant',
    'allowJoinBeforeHost' => true,
    'startHostVideo' => true,
    'startParticipantsVideo' => false
    );
try {
  $resp = $platform->post('/account/~/extension/~/meeting', $params);
  print_r ('Start Your Meeting: ' . $resp->json()->links->startUri . "\n");
  print_r ('Join the Meeting: ' . $resp->json()->links->joinUri . "\n");
} catch (Exception $e) {
  print_r ("An error occurred: " . $e->getMessage() . "\n");
}
